src/components/tiles/geometry/geometry-content.tsx

```diff
@@ -869,8 +884,39 @@ export class GeometryContentComponent extends BaseComponent<IProps, IState> {
     return this.handleDelete();
   };
 
+  private handleNewImage = (image: ImageMapEntryType) => {
+    if (this._isMounted) {
+      const content = this.getContent();
+      this.setState({ isLoading: false, imageEntry: image });
+      if (image.contentUrl && (image.contentUrl !== content.bgImage?.url)) {
+        this.updateImage(image.contentUrl, image.filename);
+      }
+    }
+  };
+
   // paste from clipboard
-  private handlePaste = () => {
+  private handlePaste = async () => {
+    // For authoring, we support the pasting of an image URL to set the background image of
+    // the geometry tile. So before pasting in the CLUE clipboard contents, we check if the
+    // system clipboard contains a text/plain value matching the expected pattern for an
+    // image URL in the CMS. If so, we paste that image URL into the geometry tile. See
+    // comment about system clipboard in the handleCopy function for more details.
+    if (navigator.clipboard.read) {
+      const osClipboardContents = await navigator.clipboard.read();
+      const osClipboardIsPlainText = !!(osClipboardContents[0].types[0] === "text/plain");
+      if (osClipboardIsPlainText) {
+        const clipboardContent = await osClipboardContents[0].getType("text/plain");
+        const clipboardContentText = await clipboardContent.text();
+        const url = clipboardContentText.match(/curriculum\/([^/]+\/images\/.*)/);
+        if (url) {
+          pasteClipboardImage(({ image }) => this.handleNewImage(image));
+        } else {
+          console.log("ERROR: invalid image URL");
+        }
+        return;
+      }
+    }
```

Comment:
I'm not sure the best way to do this, but it seems most of this should be moved into `clipboard-utils`. Then it could also match `image/png` type clipboard objects. 

I could be that pasteClipboardImage returns true or false indicating if the clipboard could be handled.

I'd also recommend putting more info in the console log so it is easier to know why we are seeing this if someone randomly pastes text in the wrong place.